### PR TITLE
Feature/additional fund fields

### PR DESCRIPTION
**Pull request by denysburnis**

### CODE REVIEW

File: controllers/fund.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def renderFund(self, fund_id):
                     x = [x for x in range(6)]
 
         return {
-            'fund_name': fund.fund_name,
+            'fund_name': fund.fund_name + ", fund_vintage: " + str(fund.fund_vintage),
```
Comment (dveb8886) [inline controllers/fund.py:44]: This looks like temporary stuff that should be removed

File: controllers/fund.py [hunk 2]
```diff
@@ -51,8 +51,8 @@ def renderFund(self, fund_id):
         }
 
     # This function creates a new fund with default stats
-    def addFund(self, fund_name, org_id):
-        fund = Fund.add(fund_name, org_id)
+    def addFund(self, fund_name, fund_manager, fund_vintage, fund_nav, fund_unfunded, prof):
+        fund = Fund.add(fund_name, fund_manager, fund_vintage, fund_nav, fund_unfunded, prof)
```
Comment (dveb8886) [inline controllers/fund.py:55]: good catch with org_id 👍

File: templates/organization.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
 </head>
 <body>
     <h1>{{ org.org_name }}</h1>
-    <h4>Profiles</h4>
+    <h4>Portfolios</h4>
```
Comment (dveb8886) [inline templates/organization.html:13]: We should rename this everywhere

File: templates/profile.html [hunk 4]
```diff
@@ -14,11 +14,30 @@ <h4>Funds</h4>
     <div>
         <ol>
         {% for fund in funds %}
-            <li><a href="{{ url_for('fund_api.fund', fund_id=fund.id) }}">{{ fund.fund_name }}</a></li>
+            <li><a href="{{ url_for('fund_api.fund', fund_id=fund.id) }}">{{ fund.fund_manager }} > {{ fund.fund_name }} > {{ fund.fund_vintage }} > {{ fund.fund_nav }} > {{ fund.fund_unfunded }}</a></li>
```
Comment (dveb8886) [inline templates/profile.html:17]: This looks temporary and should be removed

**Review by dveb8886 — CHANGES_REQUESTED** [review]

File: templates/profile.html (hunk 4 above)
Comment (denysburnis) [reply templates/profile.html:17]: This is a preparation for a new page look.

File: controllers/fund.py (hunk 1 above)
Comment (dveb8886) [reply controllers/fund.py]: This looks good

File: templates/organization.html (hunk 3 above)
Comment (dveb8886) [reply templates/organization.html:13]: We'll handle this another day

File: templates/profile.html (hunk 4 above)
Comment (dveb8886) [reply templates/profile.html:17]: Agreed

**Review by dveb8886 — APPROVED** [review]

**Pull request merged**